src/Symfony/Component/Console/Tests/Helper/SymfonyQuestionHelperTest.php

```diff
@@ -101,6 +103,15 @@ public function testAskEscapeLabel()
         $this->assertOutputContains('Do you want a \?', $output);
```

Comment:
As I said, your fix is legitimate, but there is still another issue elsewhere.

That said, the test mentioned does not actually cover anything I think and should be removed ^^'